===== Patch instructions generated by SAVER =====
[+] { Insert: if (true) free(*(swTableColumn_add:col->name)) at 8 (line 126, column 9),
===== Actual patch in diff format =====
--- a/table1/benchmarks/swoole-src/src/memory/Table.c
+++ b/table1/benchmarks/swoole-src/src/memory/Table.c
@@ -122,6 +122,8 @@ int swTableColumn_add(swTable *table, char *name, int len, int type, int size)
         col->type = SW_TABLE_STRING;
         break;
     default:
+        free(col->name->str);
+        free(col->name);
         swWarn("unkown column type.");
         return SW_ERR;
     }
